### PR TITLE
Improving build speed

### PR DESCRIPTION
**Pull request by philsturgeon**
By pinning everything to node v12 and updating the webpack targets to match,
and by removing monaco and other spectral sandbox dependencies, we've shaved some time off this build.

### CODE REVIEW

**Comment by lottamus** [comment]
:heavy_check_mark: Deploy preview for *stoplightio* ready!


:hammer: Explore the source changes: c7fd5ad5a3e686b805b705db3ed5b9ac2d66fc5d

:mag: Inspect the deploy logs: [https://app.netlify.com/sites/stoplightio/deploys/5ff229ca97fa460007779649](https://app.netlify.com/sites/stoplightio/deploys/5ff229ca97fa460007779649?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-1703--stoplightio.netlify.app](https://deploy-preview-1703--stoplightio.netlify.app?utm_source=github&utm_campaign=bot_dp)

**Comment by lottamus** [comment]
:heavy_check_mark: Deploy preview for *stoplightio* ready!


:hammer: Explore the source changes: ff0629f40d5235ad95237cd05ede74c42ffaded4

:mag: Inspect the deploy logs: [https://app.netlify.com/sites/stoplightio/deploys/5ff229e7746bc50007b30210](https://app.netlify.com/sites/stoplightio/deploys/5ff229e7746bc50007b30210?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-1703--stoplightio.netlify.app](https://deploy-preview-1703--stoplightio.netlify.app?utm_source=github&utm_campaign=bot_dp)

File: package.json [hunk 1]
```diff
@@ -9,31 +9,27 @@
...
   "dependencies": {
     "@sentry/browser": "^5.22.3",
     "@sentry/tracing": "^5.22.3",
-    "@stoplight/monaco": "^4.4.0",
     "@stoplight/react-error-boundary": "^1.0.0",
-    "@stoplight/spectral": "^4.0.0",
     "@tippyjs/react": "^4.0.2",
-    "ajv": "^6.7",
     "ansi-styles": "4.x.x",
-    "axios": "^0.19.2",
```
Comment (philsturgeon) [inline package.json:36]: This is a security bump, not that it probably matters in our mostly static site, but had to shut up the warnings.

File: tsconfig.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
   "exclude": ["node_modules", "build", "scripts", "netlify", "tmp"],
   "compilerOptions": {
     "baseUrl": ".",
-    "module": "esnext",
```
Comment (philsturgeon) [inline tsconfig.json:6]: Now that we're on Node 12 this is the recommended config that skips loads of polyfills we dont need.

**Review by paulatulis — APPROVED** [review]

**Comment by philsturgeon** [comment]
Thank you @paulatulis !

**Pull request merged**